great_expectations/core/usage_statistics/util.py

```diff
@@ -29,6 +20,10 @@ def send_usage_message(
 
         event_payload.update({"api_version": api_version})
 
+        from great_expectations.core.usage_statistics.usage_statistics import (
+            send_usage_message as send_usage_stats_message,
+        )
+
```

Comment:
As we've moving all our logic to the central `Anonymizer` class, cyclic import issues are bountiful.

I've opted to make these imports in-line so any usage of util functions here will not raise errors elsewhere in the module. Don't love it but seems to be necessary with the other changes made herein.